Source/projections/Internal/ProjectionProcessor.ts

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) Dolittle. All rights reserved.
```

Comment:
This class needs a once over with documentation

Reply:
I added some docs to the class and constructor